config/v1/types_feature.go

```diff
@@ -97,44 +97,98 @@ type FeatureGateEnabledDisabled struct {
 //
 // If you put an item in either of these lists, put your area and name on it so we can find owners.
 var FeatureSets = map[FeatureSet]*FeatureGateEnabledDisabled{
-	Default: {
-		Enabled: []string{
-			"RotateKubeletServerCertificate", // sig-pod, sjenning
-			"SupportPodPidsLimit",            // sig-pod, sjenning
-			"NodeDisruptionExclusion",        // sig-scheduling, ccoleman
-			"ServiceNodeExclusion",           // sig-scheduling, ccoleman
-			"SCTPSupport",                    // sig-network, ccallend
-		},
-		Disabled: []string{
-			"LegacyNodeRoleBehavior", // sig-scheduling, ccoleman
-		},
-	},
+	Default: defaultFeatures,
 	CustomNoUpgrade: {
 		Enabled:  []string{},
 		Disabled: []string{},
 	},
-	TechPreviewNoUpgrade: {
-		Enabled: []string{
-			"RotateKubeletServerCertificate", // sig-pod, sjenning
-			"SupportPodPidsLimit",            // sig-pod, sjenning
-			"NodeDisruptionExclusion",        // sig-scheduling, ccoleman
-			"ServiceNodeExclusion",           // sig-scheduling, ccoleman
-		},
-		Disabled: []string{
-			"LegacyNodeRoleBehavior", // sig-scheduling, ccoleman
-		},
+	TechPreviewNoUpgrade: newDefaultFeatures().toFeatures(),
```

Comment:
`SCTPSupport` was not previously part of `TechPreviewNoUpgrade` feature set?

Reply:
> `SCTPSupport` was not previously part of `TechPreviewNoUpgrade` feature set?

Yeah, it was bugged.